projects/1.19/assets/dimensional-doors/dimdoors/lang/zh_cn.json

```
@@ -0,0 +1,426 @@
+  "block.dimdoors.gray_ancient_fabric": "灰色远古之壁",
+  "block.dimdoors.silver_ancient_fabric": "浅灰色远古之壁",
+  "block.dimdoors.cyan_ancient_fabric": "青色远古之壁",
+  "block.dimdoors.purple_ancient_fabric": "紫色远古之壁",
+  "block.dimdoors.blue_ancient_fabric": "蓝色远古之壁",
+  "block.dimdoors.brown_ancient_fabric": "棕色远古之壁",
+  "block.dimdoors.green_ancient_fabric": "绿色远古之壁",
+  "block.dimdoors.red_ancient_fabric": "红色远古之壁",
+  "block.dimdoors.eternal_fabric": "永恒之壁",
+  "block.dimdoors.decayed_block": "衰变的方块",
+  "block.dimdoors.unfolded_block": "崩解的方块",
+  "block.dimdoors.unwarped_block": "扭曲的方块",
+  "block.dimdoors.unravelled_block": "破碎的方块",
+  "block.dimdoors.unravelled_fabric": "破碎之壁",
+  "block.dimdoors.eternal_fluid": "永恒源质",
+  "block.dimdoors.solid_static": "干扰的方块",
+  "block.dimdoors.detached_rift": "裂痕气泡",
+  "block.dimdoors.tesselating_loom": "嵌合织机",
+
+  "dimdoors.autogen_item_prefix": "%s制成的维度门",
+  "item.dimdoors.gold_door": "金门",
+  "item.dimdoors.quartz_door": "石英门",
+  "item.dimdoors.iron_dimensional_door": "铁质维度门",
+  "item.dimdoors.gold_dimensional_door": "金质维度门",
+  "item.dimdoors.quartz_dimensional_door": "私人维度门",
+  "item.dimdoors.unstable_dimensional_door": "不稳定维度门",
+  "item.dimdoors.oak_dimensional_door": "橡木维度门",
+  "item.dimdoors.rift_key": "裂痕之钥",
+  "item.dimdoors.rift_signature": "裂痕印记",
+  "item.dimdoors.rift_signature.stored": "坐标已存储！",
+  "item.dimdoors.rift_signature.created": "裂痕已创建！",
+  "item.dimdoors.stabilized_rift_signature": "稳定裂痕印记",
+  "item.dimdoors.stabilized_rift_signature.stored": "坐标已存储！",
+  "item.dimdoors.stabilized_rift_signature.created": "稳定裂痕已创建！",
+  "item.dimdoors.rift_configuration_tool": "裂痕配置器",
+  "item.dimdoors.rift_configuration_tool.info": "右击裂痕以移除其所有的绑定数据并为其分配一个地牢ID，右击除裂痕外的任意方块以显示被分配的裂痕数，左击任意方块以删除储存的地牢ID。",
+  "item.dimdoors.rift_remover": "裂痕移除器",
+  "item.dimdoors.rift_remover.closing": "这条裂痕即将关闭……",
+  "item.dimdoors.rift_remover.already_closing": "这条裂痕已经处于关闭中了",
+  "item.dimdoors.rift_stabilizer": "裂痕稳定器",
+  "item.dimdoors.rift_stabilizer.info": "对着裂痕的核心处右击以阻止其扩大",
+  "item.dimdoors.rift_stabilizer.stabilized": "这条裂痕已被稳定并将停止扩大",
+  "item.dimdoors.rift_stabilizer.already_stabilized": "这条裂痕已被稳定",
+  "item.dimdoors.rift_blade": "裂痕之刃",
+  "item.dimdoors.rift_blade.rift_miss": "你只能在裂痕的核心处使用此物品",
+  "item.dimdoors.world_thread": "世界纤维",
+  "item.dimdoors.infrangible_fiber": "坚韧纤维",
+  "item.dimdoors.frayed_filament": "崩解的纤维",
+  "item.dimdoors.stable_fabric": "稳定构造",
+  "item.dimdoors.world_thread_boots": "世界纤维织物靴",
+  "item.dimdoors.world_thread_helmet": "世界纤维织物帽子",
+  "item.dimdoors.world_thread_leggings": "世界纤维织物裤子",
+  "item.dimdoors.world_thread_chestplate": "世界纤维织物外套",
+  "item.dimdoors.rift_key.bound.info": "已绑定！",
+  "item.dimdoors.rift_key.unbound.info": "已解除绑定！",
+  "item.dimdoors.rift_key.no_links": "目前没有已绑定的维度门",
+  "block.dimdoors.iron_dimensional_door.info0": "放置在裂痕核心下方的方块",
+  "block.dimdoors.iron_dimensional_door.info1": "以进入裂痕对应的地方",
```

```suggestion
  "block.dimdoors.iron_dimensional_door.info1": "以激活此裂痕",
```
原文大概是这个意思
这里是按照实际表现翻译的吗